specification/eventgrid/data-plane/Microsoft.AgFoodPlatform/stable/2018-01-01/AzureDataManagerForAgriculture.json

```diff
@@ -0,0 +1,2245 @@
+      "properties": {
+        "managementZoneId": {
+          "description": "Management Zone Id associated with the zone.",
+          "type": "string"
+        },
+        "partyId": {
+          "description": "Id of the party it belongs to.",
+          "type": "string"
+        },
+        "actionType": {
+          "$ref": "#/definitions/ResourceActionType"
+        },
+        "status": {
+          "description": "Status of the resource.",
+          "type": "string"
+        },
+        "modifiedDateTime": {
+          "format": "date-time",
+          "description": "Date-time when resource was last modified, sample format: yyyy-MM-ddTHH:mm:ssZ.",
+          "type": "string"
+        },
+        "eTag": {
+          "description": "The ETag value to implement optimistic concurrency.",
+          "type": "string",
+          "x-ms-format": "etag"
+        },
+        "properties": {
+          "description": "A list of key value pairs that describe the resource.\r\nOnly string and numeral values are supported.",
+          "type": "object",
+          "additionalProperties": {}
+        },
+        "id": {
+          "description": "Unique id of resource.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name to identify resource.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Textual description of resource.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Date-time when resource was created, sample format: yyyy-MM-ddTHH:mm:ssZ.",
+          "type": "string"
+        }
+      },
+      "x-ms-examples": {
+        "productDeletedCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/zone_changed_v2.json"
+        },
+        "productDeletedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/zone_changed_v2.json"
+        }
+      }
+    },
+    "FarmHierarchyResourceType": {
+      "description": "Farm hierarchy resource type.",
+      "enum": [
+        "Party",
+        "Farm",
+        "Field",
+        "SeasonalField",
+        "Boundary"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "FarmHierarchyResourceType",
+        "modelAsString": true
+      }
+    },
+    "JobStatus": {
+      "description": "Various states a job can be in.",
+      "enum": [
+        "Waiting",
+        "Running",
+        "Succeeded",
+        "Failed",
+        "Cancelled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "JobStatus",
+        "modelAsString": true
+      }
+    },
+    "NutrientAnalysisParentType": {
+      "description": "Nutrient analysis parent type.",
+      "enum": [
+        "PlantTissueAnalysis"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "NutrientAnalysisParentType",
+        "modelAsString": true
+      }
+    },
+    "ResourceActionType": {
```

Comment:
Should this be something like DataManagerForAgriResourceActionType? ResourceActionType is too general.

Reply:
Suggested Prefix for all definitions is AgriFoodFarming.